packages/expo-notifications/src/scheduleNotificationAsync.ts

```diff
@@ -29,6 +40,23 @@ function parseTrigger(userFacingTrigger: NotificationTriggerInput): NativeNotifi
     return { type: 'date', timestamp: userFacingTrigger.getTime() };
   } else if (typeof userFacingTrigger === 'number') {
     return { type: 'date', timestamp: userFacingTrigger };
+  } else if (isDailyTriggerInput(userFacingTrigger)) {
+    const hour = userFacingTrigger.hour;
+    const minute = userFacingTrigger.minute;
+    if (hour === undefined || minute === undefined) {
```

Comment:
Yea, I think that may have been an oversight / incorrect thinking. I was thinking the `null` case would fail the range check below but `undefined` would be zero. 

The `in` check only checks for the key. So I would think that having `undefined` or `null` for the value of a key that is being checked woudl still be true. 

Let me add `null` to the conditional as well.